packages/cli/src/commands/eject/index.ts

```diff
@@ -0,0 +1,175 @@
+
+export function ejectRoutes(flags: Arguments) {
+  if (flags._[1] === 'help' || flags.help || flags.h) {
+    printHelp({
+      commandName: `${pkg.name} eject`,
+      usage: '[folder] [...options]',
+      tables: {
+        Options: [
+          [
+            '--force',
+            `Overwrite existing files in the target directory without prompting (default ${chalk.yellow(DEFAULT_VALUES.force)})`,
+          ],
+        ],
+      },
+    });
+
+    return 0;
+  }
+
+  try {
+    return _eject(flags);
+  } catch (error) {
+    console.error(`${errorLabel()} Command failed`);
+
+    if (error.stack) {
+      console.error(`\n${error.stack}`);
+    }
+
+    process.exit(1);
+  }
+}
+
+async function _eject(flags: EjectOptions) {
+  let folderPath = flags._[1] !== undefined ? String(flags._[1]) : undefined;
+
+  if (folderPath === undefined) {
+    folderPath = process.cwd();
+  } else {
+    folderPath = path.resolve(process.cwd(), folderPath);
+  }
+
+  /**
+   * First we make sure that the destination has the correct files
+   * and that there won't be any files overwritten in the process.
+   *
+   * If there are any and `force` was not specified we abort.
+   */
+  const { astroConfigPath, srcPath, pkgJsonPath, astroIntegrationPath, srcDestPath } = validateDestination(
+    folderPath,
+    flags.force,
+  );
+
+  /**
+   * We proceed with the astro configuration.
+   *
+   * There we must disable the default routes so that the
+   * new routes that we're copying will be automatically picked up.
+   */
+  const astroConfig = await parseAstroConfig(astroConfigPath);
+
+  replaceArgs({ defaultRoutes: false }, astroConfig);
+
+  fs.writeFileSync(astroConfigPath, generateAstroConfig(astroConfig));
+
+  /**
+   * We copy all assets from the `default` folder into the `src` folder.
+   */
+  fs.cpSync(srcPath, srcDestPath, { recursive: true });
+
+  /**
+   * Last, we ensure that the package.json contains the extra dependencies.
+   * If any are missing we suggest to install the new dependencies.
+   */
+  const pkgJson: PackageJson = JSON.parse(fs.readFileSync(pkgJsonPath, 'utf-8'));
+  const astroIntegrationPkgJson: PackageJson = JSON.parse(
+    fs.readFileSync(path.join(astroIntegrationPath, 'package.json'), 'utf-8'),
+  );
+
+  let hasChanged = false;
+
+  for (const dep of REQUIRED_DEPENDENCIES) {
+    if (!(dep in pkgJson.dependencies) && !(dep in pkgJson.devDependencies)) {
+      pkgJson.dependencies[dep] = astroIntegrationPkgJson.dependencies[dep];
+
+      hasChanged = true;
+    }
+  }
+
+  updateWorkspaceVersions(pkgJson.dependencies, TUTORIALKIT_VERSION, (dependency) =>
+    REQUIRED_DEPENDENCIES.includes(dependency),
+  );
+
+  if (hasChanged) {
+    fs.writeFileSync(pkgJsonPath, JSON.stringify(pkgJson, undefined, 2), { encoding: 'utf-8' });
+
+    console.log(primaryLabel('INFO'), 'New dependencies added, install the new dependencies before proceeding');
```

Comment:
We could indeed to it for them but I kept it simple for now.

If there are no `node_modules` then the `eject` command won't work because it moves the files of the version of `@tutorialkit/astro` that they use (instead of using a possibily incompatible one).